packages/manager/apps/telecom/src/app/telecom/telephony/index.js

```diff
@@ -25,6 +26,14 @@ angular
           resolve: {
             breadcrumb: /* @ngInject */ ($translate) =>
               $translate.instant('telephony_breadcrumb'),
+            isSvaWalletFeatureAvailable: /* @ngInject */ (
+              TelephonySvaWalletService,
+            ) => TelephonySvaWalletService.isFeatureAvailable(),
+            isSvaWalletValid: /* @ngInject */ (
+              TelephonySvaWalletService,
+            ) => () => TelephonySvaWalletService.isSvaWalletValid(),
```

Comment:
Maybe add test on `isSvaWalletFeatureAvailable` to avoid request when feature is not available ?

Reply:
👍